api/host.go

```diff
@@ -22,10 +22,70 @@ const (
 	UsabilityFilterModeUnusable = "unusable"
 )
 
+var validHostSortBy = map[string]any{
```

Comment:
`nit` I'd sort these alphabetically

Reply:
Oooops I actually wanted to get rid of the sorting altogether. Seems like I double reverted. Pushing that feature back to v2 for when the price tables becomes simpler.